sites/fast-frame/webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const HtmlWebpackPlugin = require("html-webpack-plugin"); // Require  html-webpack-plugin plugin
+
+module.exports = {
+    entry: __dirname + "/src/app/index.js", // webpack entry point. Module to start building dependency graph
+    output: {
+        path: __dirname + "/dist", // Folder to store generated bundle
+        filename: "bundle.js", // Name of generated bundle after build
+        publicPath: "/", // public URL of the output directory when referenced in a browser
+    },
+    module: {
+        // where we defined file patterns and their loaders
+        rules: [
+            {
+                test: /.tsx?$/,
+                use: [
+                    {
+                        loader: "ts-loader",
+                    },
+                ],
+            },
+            {
+                test: /\.js$/,
```

Comment:
I don't expect this project to have a js that isn't in `node_modules` so I'm not sure this is necessary